Gopkg.toml

```diff
@@ -42,7 +42,7 @@
 
 [[constraint]]
   name = "github.com/urfave/cli"
-  version = "1.20.0"
+  revision = "e6cf83ec39f6e1158ced1927d4ed14578fda8edb"
```

Comment:
Please put a comment to clarify why we depend on specific revision.

I guess this is some dependency related error, but it is best to have it documented.

Reply:
Added comment.

Reason: Urfave CLI uses go mod, and some dependencies were not resolved.